class/Defaults/Trigger/WordPress/ActivePlugin.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * WordPress Active Plugin trigger
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Defaults\Trigger\WordPress;
```

Comment:
I'm thinking about the file organization and because we'll have a bunch of triggers for plugins and themes, maybe will be better to place it under `BracketSpace\Notification\Defaults\Trigger\Plugin` and then make the class name just `Activated`